base_phone/models/phone_common.py

```diff
@@ -5,11 +5,15 @@
 from openerp import models, api
 from openerp.addons.base_phone.fields import Phone
```

Comment:
Thank you, @lasley. Now, I believe in the coming Travis run, the last pieces are Javascript and input on the get_caller_name.py on how to break up a literal configuration example as it runs over the 79 characters.